package.json

```diff
@@ -42,6 +42,7 @@
     "babel-plugin-add-module-exports": "^1.0.0",
     "babel-plugin-istanbul": "^5.1.0",
     "babel-plugin-transform-runtime": "^6.23.0",
+    "babel-polyfill": "^6.26.0",
```

Comment:
Why did you add this package?

Reply:
I checked in with Priscilla and Theo and they said they put it there to enable the async/await syntax

Reply:
They also said that the package is not needed again. So I'm guessing i should remove it.